frame/treasury/src/lib.rs

```diff
@@ -44,6 +44,17 @@
 //! countdown period, the median of all declared tips is paid to the reported beneficiary, along
 //! with any finders fee, in case of a public (and bonded) original report.
 //!
+//! ### Bounty
+//!
+//! A Bounty Spending is a reward for a specified body of work - or specified set of objectives - that
+//! needs to be executed for a predefined Treasury amount to be paid out. A Curator is assigned to be delegated
+//! the responsibility of assigning a payout address once the specified set of objectives is completed.
+//!
+//! After the Council has activated a bounty, it delegates the work that requires expertise to the Curator. They
+//! get to close the Active bounty. Closing the Active bounty enacts a delayed payout to the payout address. The
+//! delay allows for intervention through regular democracy.
```

Comment:
Regular democracy might be too slow. Why not intervention by a majority of the council?

Reply:
Also, what is the process for intervening? Is there a specific dispatch to call? Does the (sub)bounty get completely cancelled? Or just the payment?